cfgov/unprocessed/css/blog-docs.less

```diff
@@ -5,207 +5,29 @@
 
 
 /* topdoc
-  name: Slugs
-  family: cfgov-blog-docs
-  patterns:
-    - name: Blog slugs
-      markup: |
-        <a href="#" class="category-slug">
-            <span class="category-slug_icon cf-icon cf-icon-bullhorn"></span>
-            <span class="u-visually-hidden">Category:</span>
-            Announcements & updates
-        </a><br>
-        <a href="#" class="category-slug">
-            <span class="category-slug_icon cf-icon cf-icon-information"></span>
-            <span class="u-visually-hidden">Category:</span>
-            Consumer information
-        </a><br>
-        <a href="#" class="category-slug">
-            <span class="category-slug_icon cf-icon cf-icon-dialogue"></span>
-            <span class="u-visually-hidden">Category:</span>
-            Engagement
-        </a><br>
-        <a href="#" class="category-slug">
-            <span class="category-slug_icon cf-icon cf-icon-lightbulb"></span>
-            <span class="u-visually-hidden">Category:</span>
-            Innovation & data
-        </a>
-      notes:
-        - "Official icons are still being developed, hence the ?."
+    name: Press Information
+    markup: |
+      <div class="press-information">
+          <h1 class="header-slug">
+              <span class="header-slug_inner">
+                  Press information
+              </span>
+          </h1>
+          <p class="short-desc">
+              If you want to repbulish the article or have questions about the content,
+              please contact the press office.
+          </p>
+          <p>
+              <a class="sans"
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3063/commits/7300ea4e98231afe4a6ccfb538242ac2c040656d